server/channels/api4/shared_channel_test.go

```diff
@@ -165,7 +165,12 @@ func TestCreateDirectChannelWithRemoteUser(t *testing.T) {
 		th := setupForSharedChannels(t).InitBasic()
 		defer th.TearDown()
 		client := th.Client
-		defer client.Logout(context.Background())
+		defer func() {
+			_, err := client.Logout(context.Background())
+			if err != nil {
+				t.Fatal(err)
+			}
```

Comment:
Please use `require.NoError(t, err)` instead of `t.Fatal` here and in the other places

Reply:
Resolved!!!